keyboards/anavi/macropad10/keyboard.json

```
@@ -17,6 +17,11 @@
         "nkro": true,
         "rgblight": true
     },
+    "backlight": {
+        "pin": "GP26",
+        "driver": "pwm",
+        "levels": 3
```

`"levels": 3` duplicates defaults. The `software` driver is mostly equivalent to the code you initially submitted, though correctly configuring `pwm` would be better.

```suggestion
        "driver": "software"
```